modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/DriverInfo/DriverInfoForm.tsx

```diff
@@ -553,59 +599,103 @@ const DriverInfoForm = ({
 						</ClaySelect>
 
 						<label htmlFor="occupation">
-							Occupation&nbsp;
+							Ocuppation&nbsp;
```

Comment:
Please takeout the extra 'p' and add back 'c'
```suggestion
							Occupation&nbsp;
```